### ISSUE TITLE
Flutter v1.5.9-pre.108 - master issue

### ISSUE THREAD

**Issue by calvinmuller** [Sh1d0w/multi_image_picker#135]
**Bug Report**

`multi_image_picker` version: 4.2.0

Is it happening on Android: Yes

Is it happening on iOS:  Yes

```
[✓] Flutter (Channel master, v1.5.9-pre.108, on Mac OS X 10.14.4 18E226, locale en-ZA)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
[✓] Android Studio (version 3.3)
[✓] IntelliJ IDEA Ultimate Edition (version 2018.3.2)
[✓] Connected device (1 available)

• No issues found!

```

**Describe the bug**

Building a project results in 
```
file:///Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/multi_image_picker-4.2.0/lib/src/asset_thumb_provider.dart:35:31: Error: The argument type 'Null Function(StringBuffer)' can't be assigned to the parameter type 'Iterable<DiagnosticsNode> Function()'.
 - 'StringBuffer' is from 'dart:core'.
 - 'Iterable' is from 'dart:core'.
 - 'DiagnosticsNode' is from 'package:flutter/src/foundation/diagnostics.dart' ('file:///Users/calvinmuller/iStreet/Mobile/flutter/packages/flutter/lib/src/foundation/diagnostics.dart').
Try changing the type of the parameter, or casting the argument to 'Iterable<DiagnosticsNode> Function()'.
        informationCollector: (StringBuffer information) {
                              ^
```

**Comment by Sh1d0w** [comment]
Thanks for the report.

This plugin supports only the stable version of Flutter, which currently is 1.2.1.

I can't guarantee it will work properly on nightly builds, as they are release every day, usually with breaking changes, also a lot of stuff gets reverted back.

I would advise you to stick to the stable version if you want stability.

**Comment by Sh1d0w** [comment]
I've released version 4.2.1 which should solve the problem.

Again, it is good practice to use only stable version of Flutter.
